molecule/command/create.py

```diff
@@ -105,7 +105,7 @@ def execute(self):
 @click.option(
     '--driver-name',
     '-d',
-    type=click.Choice(drivers()),
+    type=click.Choice([str(s) for s in drivers()]),
```

Comment:
Almost feel like we should reference the issue here in a comment but I leave it to you.